addon/lib/Feature.jsm

```diff
@@ -97,24 +97,28 @@ class Feature {
       [{
         label: "Thanks!",
         isDefault: true,
-        acceptButton() {
+        callback: function acceptButton() {
+          console.log("clicked THANKS!");
           feature.telemetry({
             event: "introduction-accept",
           });
         },
       },
       {
         label: "I do not want this.",
-        leaveStudyButton() {
+        callback: function leaveStudyButton() {
+          console.log("clicked NO!");
           feature.telemetry({
             event: "introduction-leave-study",
           });
-          feature.studyUtils.endStudy("");
+          feature.studyUtils.endStudy("introduction-leave-study");
         },
       }],
       // callback for nb events
       null
     );
+
+    notice.setAttribute("data-study-config", JSON.stringify(this.variation));
```

Comment:
This looks to be exclusively for testing -- perhaps note that?

Reply:
Done.